js/hollaex.js

```diff
@@ -953,10 +953,11 @@ module.exports = class hollaex extends Exchange {
         await this.loadMarkets ();
         const request = {};
         let market = undefined;
-        if (symbol !== undefined) {
-            market = this.market (symbol);
-            request['symbol'] = market['id'];
+        if (symbol === undefined) {
```

Comment:
If possible, please move this check above `loadMarkets()` to throw it early.

Reply:
sure